client/reader/stream/style.scss

```diff
@@ -385,6 +385,12 @@
 	}
 }
 
+body.is-section-reader div.layout.is-global-sidebar-visible .is-site-stream .back-button button.is-compact.is-borderless {
+	@media ( max-width: 1300px ) {
+		margin-bottom: 24px;
+	}
+}
+
```

Comment:
We may want to take another pass at consolidating the styles here and other parts of the Reader so they're just defined once and consistent. I think we can just fix this to improve the UX right now and handle a potential refactor separately, though.